src/nets/e2e_asr_attctc.py

```diff
@@ -606,7 +619,13 @@ def recognize_beam(self, h, recog_args, char_list):
                 hyp['a_prev'] = att_w
 
                 # get nbest local scores and their ids
-                local_scores = F.log_softmax(self.output(z_list[-1])).data
+                if rnnlm:
+                    rnnlm_state, z_rnnlm = rnnlm.predictor(hyp['rnnlm_prev'], hyp['yseq'][i])
+                    hyp['rnnlm_prev'] = rnnlm_state
+                    local_scores = (1 - recog_args.lm_weight) * F.log_softmax(self.output(z_list[-1])).data \
+                        + recog_args.lm_weight * F.log_softmax(z_rnnlm).data
```

Comment:
I didn't apply (1 - lm_weight) to ATT score in our Interspeech paper because the score was already combined with CTC score as (1-lambda) * ATT + lambda * CTC and I thought that (1-lm_weight) would complicate the equation and parameter tuning. What do you think?

Reply:
I agree with you.
I'll test it.